python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -133,6 +171,13 @@ def acceptNavigationRequest(self, frame, request, n_type): # noqa
         self._view.page().networkAccessManager().authenticationRequired.connect(self.on_authentication_required)
         self._view.loadFinished.connect(self.on_load_finished)
 
+        # We want to inject custom JavaScript code before any code is
+        # executed in the loaded web pages. This is to polyfill any
+        # missing functionality.
+        frame = self._view.page().currentFrame()
+        frame.javaScriptWindowObjectCleared.connect(self._polyfill)
```

Comment:
can we include a debug message here to make it easier to support and troubleshoot?

Reply:
Added the code.